### DISCUSSION TITLE
Where can I post show-and-tell for this community?

### DISCUSSION THREAD

**Discussion by maxpert** [keystonejs/keystone#7959] [Questions] [answered]
Hey folks,

I've been trying to look through GitHub and your community channels on where I can post a show and tell for the broader keystone community. I am working on a project called [Marmot](https://github.com/maxpert/marmot) which allows you to have masterless replication and eventually consistent store for anyone using SQLite without modifying code of the application. I had a [demo video](https://youtu.be/GQ5x8pc9vuI) that I wanted to share with Keystone community to get some feedback. What is the recommended channel for doing so?

**Comment by flexdinesh** [comment, marked as answer]
We have a [Keystone Community Slack](https://community.keystonejs.com/) and there's a **#showcase** channel just for this. You'll get community feedback if you post your link there.

Use this link to join the channel: [Invite to join Keystone slack](https://slack.keystonejs.com/)

Link to channel: https://keystonejs.slack.com/archives/C01SKLJJN85

**Reply by maxpert** [reply]
Seems like I can't get in due to requirement of work-email

**Reply by flexdinesh** [reply]
You need an email address to sign up to any slack workspace. It doesn't have to be work email. It could be any email address.

**Reply by maxpert** [reply]
Sorry for confusion I am getting this when I try to get in with my email account. 
![image](https://user-images.githubusercontent.com/22441/192685012-208bac61-deb7-431d-aca3-d6126f6beca1.png)

**Reply by gautamsi** [reply]
you need to create an account for every workspace on slack. click this link
![image](https://user-images.githubusercontent.com/5769869/192687573-69e90c92-bbcd-4b65-b507-cea18f3157d9.png)

**Reply by flexdinesh** [reply]
My bad. I shared the wrong link. Instead of sharing the invite link I shared the workspace link. I've fixed it in the original response. Here's the invite link — Use this link to join the channel: [Invite to join Keystone slack](https://slack.keystonejs.com/).

Sorry about the mishap.

**Reply by kerbe** [reply]
@flexdinesh there isn't way to use email to sign up for KeystoneJS slack? That latest link you have given, shows that only email addresses that are allowed to be used to register there are keystonejs.com and thinkmill.com.au. But I can't use my own email... Is there workaround, or is only option to create either Google account or Apple account, which maybe lets you enter freely?

**Reply by flexdinesh** [reply]
We fixed the link today @kerbe as reported from this issue. Please try again now, it should work. 
- #8090
